.github/workflows/update_enigma_assessments.yaml

```
@@ -0,0 +1,60 @@
+name: Update ENIGMA-PD assessment terms JSON file
+
+on:
+  schedule:
+    # Run once a week on Monday at 8 AM EST in winter, 9 AM EDT in summer
+    - cron: '0 13 * * 1'
+  workflow_dispatch:
+
+jobs:
+  update-terms:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate a Neurobagel Bot token
+        id: generate-token
+        uses: actions/create-github-app-token@v2
+        with:
+          app-id: ${{ vars.NB_BOT_ID }}
+          private-key: ${{ secrets.NB_BOT_KEY }}
+
+      - name: Checkout
+        uses: actions/checkout@v4
```

```suggestion
        uses: actions/checkout@v5
```
to avoid the next dependabot